core/src/main/scala/dagr/core/cmdline/parsing/ParsingUtil.scala

```diff
@@ -124,7 +95,7 @@ private[parsing] object ParsingUtil {
     }
 
     // Finally, make the map
-    classes.map(clazz => Tuple2(clazz, getAnnotation(clazz, classOf[CLPAnnotation]))).toMap
+    classes.map(clazz => Tuple2(clazz, ReflectionUtil.findJavaAnnotation(clazz, classOf[CLPAnnotation]).get)).toMap
```

Comment:
It's personal preference.  So long as it's not super long, I prefer to see the object name at the usage site, to give me a clue where the symbol is coming from without having to check.